modules/unsupported/geojsonstore/src/main/java/com/bedatadriven/jackson/datatype/jts/GeoJson.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2020, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+/*
+ * Original code at https://github.com/bedatadriven/jackson-datatype-jts Apache2 license
```

Comment:
I was going to ask you to include the header from the file you are copying .... but they are empty!

With that in mind we can follow apache's guidance and write the darn header for this project in order to clearly credit:
```
/*
Copyright 2016 BeDataDriven B.V.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/
```

Reference:
* http://www.bedatadriven.com/company/contact.html
* https://www.apache.org/licenses/LICENSE-2.0.html#apply
* https://infra.apache.org/apply-license.html